uaclient/api/data_types.py

```diff
@@ -31,10 +30,16 @@ class APIData(DataObject):
         Field("meta", DataObject),
     ]
 
-    def __init__(self, *, type: str, attributes: DataObject, meta: DataObject):
+    def __init__(self, *, type: str, attributes: DataObject, meta: dict):
```

Comment:
We have two `meta` keys in our response. The top level one and the one inside `data`. Since this is a generic information, I think we should only include it on the top level `meta` field. This will also make it consistent with errors generated by the API

Reply:
Actually there is no top-level `meta` - there is one `meta` for the Data, and one for each Error/Warning.
Do you think we should unify those as a single top-level key meta?